docs/user-guide/cli-installcli.md

```diff
@@ -35,6 +37,8 @@ If you do not have internet access at your site, use the following method to ins
   Npm is included with most Node.js installations. Issue the command `npm --version` to check your current version. You can reference the [Node.js release matrix](https://nodejs.org/en/download/releases/) to verify that the versions are compatible.
   :::
 
+- **(Optional) ssh2 Package:** Zowe CLI has a dependency on the ssh2 package. This package allows for more secure cryptographic ciphers to be used first on supporting hardware. See [Installing the ssh2 Package for Zowe CLI](cli-installing-ssh2-package.md).
```

Comment:
This optional step should probably also be present in the other method to install Zowe CLI (around line 104)